src/gui/updater/ocupdater.cpp

```diff
@@ -487,6 +480,57 @@ void NSISUpdater::showUpdateErrorDialog(const QString &targetVersion)
     msgBox->open();
 }
 
+void NSISUpdater::showNewUpdateToInstallDialog()
+{
+    const auto name = Theme::instance()->appNameGUI();
+
+    auto updateToInstallDialog = new QDialog;
```

Comment:
Why the QDialog instead of the message box? Why did you not create a QMessageBox on the heap?

Reply:
@FlexW Fixed!